ledger/streaming/api/src/response.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use crate::{streaming_blocks::SubscribeResponse, BlockStreamComponents, Result};
+use mc_crypto_keys::{Ed25519Pair, Ed25519Public, Ed25519Signature, Signer, Verifier};
+use protobuf::Message;
+use std::convert::TryFrom;
+
+/// Creates a signed SubscribeResponse from the given components and signer.
+pub fn make_subscribe_response(
+    data: &BlockStreamComponents,
+    signer: &Ed25519Pair,
```

Comment:
I think ultimately we want to find out how often this changes, I've heard slightly differing things. What would happen currently if the node sig changed?

Reply:
I'm not sure what you mean by node sig. This key will be for signing and verifying the `SubscribeResponse`s

Reply:
This is one possible reason to do generics now, to have a "semantic signing" element.

Reply:
I thought it was the node signature that was being passed along, I see about signing the subscribe responses. As far as the purpose of collecting blocks, ultimately client-side SCP would take care ensuring peer streams are indeed a consensus. If there's a hijacked stream in the set of peers, the SCP validation would catch it.

Reply:
After discussing offline with James, we agreed to have the consensus node sign the `SubscribeResponse`s, so I removed the `make_subscribe_response` calls from `GrpcServerSink` (in #1670 ), and switched these methods to use a digest of the `BlockStreamComponents`, rather than proto encoding.

Reply:
My point was that the validator should be the one checking signatures, not the gRPC components, so all this code should be removed.